apps/platform/utils/auth/passkeys.ts

```diff
@@ -55,7 +55,7 @@ async function generateRegistrationOptions(options: RegistrationOptions) {
     authenticatorSelection: {
       residentKey: 'preferred',
       userVerification: 'preferred',
-      authenticatorAttachment
+      authenticatorAttachment: 'platform'
```

Comment:
why not using the passed value?
making it `platfrom` will break `cross-platfrom` auth like google passkeys